### PR TITLE
More `Makefile` tweaks

### PR DESCRIPTION
**Pull request by davep**
Ensure that the `syntax` extras gets installed as part of `make setup`, and also add `make repl`[^1].

[^1]: Sure, I'm probably the only one who'll ever use it, but it's a handy thing I have within every Python project and my muscle memory fails me every time I'm in the Textual development directory and every time I keep thinking "I should add that" so here I am adding that.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,6 +85,7 @@ clean: clean-screenshot-cache clean-offline-docs
 .PHONY: setup
 setup:
 	poetry install
+	poetry install --extras syntax
```
Comment (darrenburns) [inline Makefile:88]: You only need `poetry install --extras syntax`, which does both the normal dependencies plus the extras.

Comment (davep) [reply Makefile:88]: Will it leave the normal dependencies intact of the extras fail?

Comment (darrenburns) [reply Makefile:88]: That I don't know

**Review by darrenburns — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (davep) [reply Makefile:88]: Quickly testing, fresh 3.7 environment, doing the single-line version, stuff installs but you seem to end up with a non-working venv. On the other hand the two-step approach does leave you with a working venv, which is what I was aiming for (you'll still get errors, but at least the venv will be usable).

**Pull request merged**